packages/app/src/request.ts

```diff
@@ -32,17 +37,32 @@ export const getProtocol = (req: Request, trust: Trust): Protocol => {
   return index !== -1 ? header.substring(0, index).trim() : header.trim()
 }
 
+/**
+ * @deprecated
+ *
+ * @param req
+ * @param trust
+ */
 export const getHostname = (req: Request, trust: Trust): string | undefined => {
+  const { hostname } = getHost(req, trust)
+  return hostname
+}
+
+export const getHost = (req: Request, trust: Trust): Host | undefined => {
```

Comment:
`export const getHost = (req: Request, trust: Trust): { hostname: string, port?: number } | undefined => {`

This way, you won't have to define a new type.

Reply:
what's wrong with defining the type explicitly? inlining the type only seems to obfuscate and increase the line-length

Reply:
`type Host` is not reused.

Reply:
Many functions are only used once, but that is no reason to inline them